nand2tetris.el

```diff
@@ -167,13 +173,14 @@ So it can use the builtin chips."
   (let ((filename      (file-name-base (buffer-file-name)))
         (hdl-buffer    (current-buffer))
         (tst-file      (nand2tetris//get-current-test-file))
-        (cmp-file      (nand2tetris//get-current-compare-file)))
-    (copy-file tst-file (concat "/tmp/" filename ".tst") t)
-    (copy-file cmp-file (concat "/tmp/" filename ".cmp") t)
-    (with-temp-file (concat "/tmp/" filename ".hdl")
+        (cmp-file      (nand2tetris//get-current-compare-file))
+        (temp-dir      (file-name-as-directory (make-temp-file "nand2tetris-el" t))))
```

Comment:
I see how make-temp-file has a `dir-flag` that will create a directory instead. 
Maybe adding another line with a variable that explains what that extra argument does would be more readable.

```elisp
(should-create-dir t)
(temp-dir      (file-name-as-directory (make-temp-file "nand2tetris-el" should-create-dir))))
```

It's just a thought. Please feel free to challenge this comment

Reply:
Yeah that's a good idea. I wish emacs made more use of plist-style arguments for this sort of thing, random `t` and `nil` arguments aren't exactly readable.